server/data/incidentTypeConfiguration/validation.ts

```diff
@@ -34,15 +35,26 @@ function checkStartingQuestion(config: IncidentTypeConfiguration, errors: Error[
   }
 }
 
-function checkUnknownQuestions<Q>(configGraph: Graph<Q>, errors: Error[]) {
+function checkQuestionsWithoutAnswers(config: IncidentTypeConfiguration, errors: Error[]): void {
+  Object.values(config.questions)
+    .filter(question => question.active === true)
```

Comment:
Our config format should be relatively good quality, so maybe  `.filter(question => question.active)` would be fine?

The type of active is `boolean` and typecheck should throw an error if that is not the case no?

Reply:
this is copied from *your* code

Reply:
Still, what do you think 🤔?